Source/Core/ClippingPlaneCollection.js

```diff
@@ -32,20 +34,19 @@ define([
      * @param {Plane[]} [options.planes=[]] An array of up to 6 {@link Plane} objects used to selectively disable rendering on the outside of each plane.
      * @param {Boolean} [options.enabled=true] Determines whether the clipping planes are active.
      * @param {Matrix4} [options.modelMatrix=Matrix4.IDENTITY] The 4x4 transformation matrix specifying an additional transform relative to the clipping planes original coordinate system.
-     * @param {Boolean} [options.combineClippingRegions=true] If true, the region to be clipped must be included in all planes in this collection. Otherwise, a region will be clipped if included in any plane in the collection.
+     * @param {Boolean} [options.unionClippingRegions=true] If true, the region to be clipped must be included in all planes in this collection. Otherwise, a region will be clipped if included in any plane in the collection.
```

Comment:
I think this is backwards.  If it's a UNION, the value is clipped if it lies in any clipping plane.  If it's an INTERSECT, the value is clipped only if it lies in all of the planes.

![image](https://user-images.githubusercontent.com/3451886/33894861-092d2446-df2d-11e7-8eac-1344f8465d51.png)

Reply:
Got it, my mistake, I'm swap the values. I think we should default to the same behavior though, so `union` should initially be `false`

Reply:
Yep, agreed =)